capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -38,34 +38,46 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.sql.DataSource;
 
-@WebServlet("/study_set")
+@WebServlet(urlPatterns = { "/study_set/*" })
 public class StudySetServlet extends HttpServlet {
-  private final String search_sql_statement = 
+  private final String SEARCH_SQL_STATEMENT =
     "SELECT COUNT(card.study_set_id), study_set.id, study_set.title, study_set.description, " +
     "study_set.subject, university.name, user_info.user_name FROM study_set JOIN university " +
     "ON university.id = study_set.university_id JOIN user_info ON study_set.owner_id = user_info.id " +
     "JOIN card ON card.study_set_id = study_set.id WHERE study_set.subject ILIKE ? OR university.name ILIKE ? OR " +
     "study_set.title ILIKE ? OR study_set.description ILIKE ? OR user_info.user_name ILIKE ? " +
     "GROUP BY study_set.id, university.id, user_info.id";
 
+  private final String VIEW_STUDY_SET_QUERY =
+    " SELECT card.front, card.back, study_set.title, study_set.description, study_set.subject, university.name, user_info.user_name " +
+    " FROM study_set" +
+    " JOIN university ON university.id = study_set.university_id" +
+    " JOIN user_info ON study_set.owner_id = user_info.id" +
+    " JOIN card ON card.study_set_id = study_set.id" +
+    " WHERE  study_set.id= ?" +
+    " GROUP BY card.id, study_set.id, university.id, user_info.id;";
+
   public ArrayList<HashMap<String, String>> runSqlQuery(
```

Comment:
Rename this method for consistency with `runViewStudySetSqlQuery`, e.g. the word "search" should probably be in the name